cmd/substreams/init.go

```diff
@@ -67,10 +56,47 @@ func runSubstreamsInitE(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
+	wantsAbi, err := generateSelect("Would you like to track a particular contract", []string{"yes", "no"})
+	if err != nil {
+		return fmt.Errorf("running abi prompt: %w", err)
+	}
+
+	// Default 'Bored Ape Yacht Club' contract.
+	// Used in 'github.com/streamingfast/substreams-template'
+	contract := "bc4ca0eda7647a8ab7c2061c2e118a18a936f13d"
+	if wantsAbi == "yes" {
+		contract, err = generatePrompt("Contract to track: ")
+		if err != nil {
+			return fmt.Errorf("running contract prompt: %w", err)
+		}
+	}
+
+	// Clean up given contract
+	contractHex, err := eth.NewHex(contract)
+	if err != nil {
+		return fmt.Errorf("getting contract bytes: %w", err)
```

Comment:
This should be validated while doing `generateSelect("Would you like to track a particular contract", []string{"yes", "no"})`, the library allows for a custom validator